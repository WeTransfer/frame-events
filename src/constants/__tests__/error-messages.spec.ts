import ERROR_MESSAGES from '../error-messages';

describe('Error messages', () => {
  it('should match snapshot', () => {
    expect(ERROR_MESSAGES).toMatchSnapshot();
  });
});
