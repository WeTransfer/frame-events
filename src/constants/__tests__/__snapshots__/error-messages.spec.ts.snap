// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error messages should match snapshot 1`] = `
{
  "CANT_USE_READY_COMMAND": "The 'ready' command is reserved.",
  "CANT_VALIDATE_ORIGIN": "Can't validate origin! Please add ?_origin=PARENT_HOST to the iframe source.",
  "CANT_VALIDATE_PLACEMENT": "Can't validate placement! Please add ?_placement=PLACEMENT_NAME to the iframe source.",
  "EMPTY_IFRAME": "No src found. You can't run ParentFrame on an empty iframe element.",
  "INVALID_MESSAGE_FORMAT": "The message format is invalid.",
  "NOT_DEFINED_EVENT_NAME": "Can't send an undefined event name. Make sure you add your event name first.",
}
`;
